deployment/inception.sh

```diff
@@ -33,7 +33,7 @@ az group create --name $COMMON_RESOURCE_GROUP --location $AZURE_LOCATION
 ## -------
 # Create the Azure Traffic Manager profile
 AZURE_TRAFFIC_MANAGER_PROFILE_NAME=$PROJECT_NAME-trafficmgr
-az network traffic-manager profile create --name $AZURE_TRAFFIC_MANAGER_PROFILE_NAME --resource-group $COMMON_RESOURCE_GROUP --routing-method Priority --unique-dns-name $PROJECT_NAME.microservices.cse
+az network traffic-manager profile create --name $AZURE_TRAFFIC_MANAGER_PROFILE_NAME --resource-group $COMMON_RESOURCE_GROUP --routing-method Priority --unique-dns-name $PROJECT_NAME.$PUBLIC_DOMAIN_NAME_SUFFIX
```

Comment:
Make sure to update this line with ```$PROJECT_NAME.$PUBLIC_DOMAIN_NAME_SUFFIX```

https://github.com/Microsoft/containerized-microservices-pipeline/blob/master/deployment/inception.sh#L22

Reply:
fixed